packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -224,12 +223,15 @@ class TransactionService {
         }
         this._config.events.send(TRANSACTION_END, [tr])
         if (__DEV__) {
-          this._logger.debug('TransactionService transaction ended', tr)
+          this._logger.debug(`end transaction(${tr.id}, ${tr.name})`, tr)
         }
       },
       err => {
         if (__DEV__) {
-          this._logger.debug('TransactionService transaction onEnd', err)
+          this._logger.debug(
+            `error ending transaction(${tr.id}, ${tr.name})`,
```

Comment:
nit: Capital first letters!

Reply:
I went with small since we have function names as `startTransaction` and it was a bit odd seeing only these in capital letters.